internal/cmd/backup.go

```diff
@@ -111,39 +290,307 @@ func backupCmdFunc(cmd *cobra.Command, args []string) error {
 		}
 
 		for _, rel := range relsResp.Relationships {
-			if err := encoder.Append(rel); err != nil {
-				return fmt.Errorf("error storing relationship: %w", err)
-			}
-			stored++
+			if hasRelPrefix(rel, prefixFilter) {
+				if err := encoder.Append(rel); err != nil {
+					return fmt.Errorf("error storing relationship: %w", err)
+				}
+				relsEncoded++
 
-			if stored%100_000 == 0 && !hasProgressbar {
-				log.Trace().Uint("relationships", stored).Msg("relationships stored")
+				if relsEncoded%100_000 == 0 && !isatty.IsTerminal(os.Stderr.Fd()) {
+					log.Trace().
+						Uint("encoded", relsEncoded).
+						Uint("processed", relsProcessed).
```

Comment:
nit: a percentage value is easier to parse when someone is looking at the logs

Reply:
we don't know the total number of relationships in the database